WebRoot/h/printconversations

```diff
@@ -28,8 +28,13 @@
 <body style='background:white;'>
 <div class='ZhCallListPrintView'>
     <table><tr>
-        <td><b><fmt:message key="zimbraTitle"/></b></td>
-        <c:set var="mailboxName" value="${(not empty param.acct ? param.acct : mailbox.name)}" />
+        <c:if test="${mailbox.attrs.zimbraEmailPrintoutHeader[0] eq 'TRUE'}">
```

Comment:
We can make it simple.

    <td><b><c:if test="${mailbox.attrs.zimbraEmailPrintoutHeader[0] eq 'TRUE'}"><fmt:message key="zimbraTitle"/></c:if></b></td>